benchmark/run_all_benchmarks.sh

```diff
@@ -70,42 +89,179 @@ run_spmv_benchmarks() {
 # is completed, the backups and the results are combined, and the newest file is
 # taken as the final result.
 run_solver_benchmarks() {
-    cp $1 "$1.imd" # make sure we're not loosing the original input
+    [ "${DRY_RUN}" == "true" ] && return
+    cp "$1" "$1.imd" # make sure we're not loosing the original input
     ./solver/solver --backup="$1.bkp" --double_buffer="$1.bkp2" \
-                    --executor=${EXECUTOR} --solvers="cg,bicgstab,cgs,fcg" \
+                    --executor="${EXECUTOR}" --solvers="cg,bicgstab,cgs,fcg" \
                     --max_iters=10000 --rel_res_goal=1e-6 \
```

Comment:
I think these settings should also be mentioned in the README.

Reply:
done